services/services_test.go

```diff
@@ -101,6 +101,58 @@ func TestServicesConfigError(t *testing.T) {
 		"Could not parse `health` in service myName: time: invalid duration xx")
 }
 
+func TestServicesConsulConfigEnableTagOverride(t *testing.T) {
+	jsonFragment := []byte(`[
+{
+  "name": "serviceA",
+  "port": 8080,
+  "interfaces": "eth0",
+  "health": ["/bin/to/healthcheck/for/service/A.sh", "A1", "A2"],
+  "poll": 30,
+  "ttl": 19,
+  "timeout": "1ms",
+  "tags": ["tag1","tag2"],
+  "consul": {
+	  "enabletagoverride": true
```

Comment:
We definitely should be testing for misconfigurations for both of these features too.